src/components/marker/timeline-preview.tsx

```diff
@@ -0,0 +1,132 @@
+    framePreviewImgStyle += `background-position: -${thumbnailInfo.x}px -${thumbnailInfo.y}px;`;
+    return framePreviewImgStyle;
+  }
+  return '';
+};
+
+interface TitleProps {
+  id: string,
+  iconPath: string,
+  children: VNode | string
+}
+
+const Title = (({id, iconPath, children}: TitleProps) => {
+  return (
+  <div className={styles.titleWrapper}>
+    <Icon
+      id={id}
+      height={icons.SmallSize}
+      width={icons.SmallSize}
+      viewBox={`0 0 ${icons.SmallSize} ${icons.SmallSize}`}
+      path={iconPath}/>
+    <span className={styles.title}>{children}</span>
+  </div>
+  );
+});
+
+@withText(translates)
+export class TimelinePreview extends Component<TimelinePreviewProps> {
+  private _handleClick = () => {
+    this.props.onArrowClick();
+  };
+
+  private _renderHeader() {
+    // organize the header by cuePoint types
+    const hotspots = this.props.cuePointsData.filter(cuePoint => cuePoint.type === ItemTypes.Hotspot);
+    const quizQuestions = this.props.cuePointsData.filter(cuePoint => cuePoint.type === ItemTypes.QuizQuestion);
+    const chapters = this.props.chaptersData;
+    const answerOnAir = this.props.cuePointsData.filter(cuePoint => cuePoint.type === ItemTypes.AnswerOnAir);
+
+    let quizQuestionTitle = {type: '', firstIndex: 1, lastIndex: ''};
+    if (quizQuestions.length) {
+      const reflectionPoint = quizQuestions.find(qq => qq.quizQuestionData.type === 3);
+      quizQuestionTitle = {
+        type: quizQuestions.length === 1 && reflectionPoint ? this.props.reflectionPointTranslate! : this.props.questionTranslate!,
+        firstIndex: quizQuestions[0].quizQuestionData.index + 1,
+        lastIndex: quizQuestions.length > 1 ? `-${quizQuestions[quizQuestions.length - 1].quizQuestionData.index + 1}` : ''
+      };
+    }
+
+    return (
+      <Fragment>
+        {chapters.length > 0 &&
+          <Title id={'timeline-chapter-icon'} iconPath={icons.CHAPTER_ICON}>{chapters[0].title}</Title>}
+        {hotspots.length > 0 && <Title id={'timeline-hotspot-icon'} iconPath={icons.HOTSPOT_ICON}>{this.props.hotspotTranslate!}</Title>}
+        {quizQuestions.length > 0 &&
+          <Title id={'timeline-quiz-question-icon'} iconPath={icons.QUIZ_ICON}>
+            <span>{`${quizQuestionTitle.type} ${quizQuestionTitle.firstIndex}${quizQuestionTitle.lastIndex}`}</span>
+          </Title>}
+        {answerOnAir.length > 0 &&
+          <Title id={'timeline-aoa-icon'} iconPath={icons.AOA_ICON}>{this.props.aoaTranslate!}</Title>}
+      </Fragment>
+    );
+  }
+
+  private _renderArrowButton() {
+    const isNavigationPluginOpen = this.props.isNavigationPluginOpen();
+    return (
+      <Fragment>
+        <button className={[styles.markerLink, !isNavigationPluginOpen ? styles.disabled : ''].join(' ')} onClick={this._handleClick}>
+          <Icon id={'timeline-marker-link-close'} height={18} width={18} viewBox={`0 0 ${24} ${24}`} path={icons.PREVIEW_HEADER_ARROW_BUTTON_CLOSE}/>
```

Comment:
I think we can put icon in `common` repo and use icon from there instead of use keep path inside `playkit-js-timeline` repo